src/d3a/models/myco_matcher/base_matcher.py

```diff
@@ -0,0 +1,24 @@
+from abc import ABC, abstractmethod
+
+
+class BaseMatcher(ABC):
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def sort_by_energy_rate(obj, reverse_order=False):
+        if reverse_order:
+            # Sorted bids in descending order
+            return list(reversed(sorted(
+                obj.values(),
+                key=lambda b: b.energy_rate)))
+
+        else:
+            # Sorted bids in ascending order
+            return list(sorted(
+                obj.values(),
+                key=lambda b: b.energy_rate))
```

Comment:
```suggestion
            return list(sorted(
                obj.values(),
                key=lambda b: b.energy_rate, reversed=reverse_order))

```

Reply:
```suggestion
        return list(sorted(obj.values(), key=lambda b: b.energy_rate, reverse=reverse_order))
```
Adapted with slight modification.

Reply:
Sorry, just tested, neither mine nor your suggestion worked.

Reply:
@faizan2590 I tested this and it worked fine

Reply:
+ I suggest we change this function to be a bit more generic, to sort depending on any attribute the user sends
For example it could be :
```python
    def sort_bids_offers(obj, attribute,  reverse_order=False):
       if hasattr(obj, attribute):
             return list(sorted(
                obj.values(),
                key=lambda b: getattr(b, attribute), reverse=reverse_order))
```

Reply:
Im not sure, how you tested but its still not working for me.

https://user-images.githubusercontent.com/33482427/117308799-4a05f380-ae9b-11eb-9573-3cfa2c29231a.mov